roles/jd-client/src/args.rs

```diff
@@ -19,10 +19,15 @@ enum ArgsResult {
 
 impl Args {
     const DEFAULT_CONFIG_PATH: &'static str = "jdc-config.toml";
+    const HELP_MSG: &'static str = "Usage: -h/--help, -c/--config <path|default jdc-config.toml>";
```

Comment:
Here it should be:
`"Usage: -h/--help, -- -c/--config <path|default jdc-config.toml>"`

The double -- is needed before indicating the config

Reply:
are you sure?

that is usually a pattern on `cargo run`, for example `cargo run -- -c jdc-config.toml`.

However this config message makes no assumptions about `cargo` vs `./target/debug/jd_client` vs `/bin/jd_client`.

f we introduce `--` for help message on any case, we mislead users of `./target/debug/jd_client` and `/bin/jd_client` approaches.

If `cargo` is used, it is assumed that the user knows he needs an extra `--` before adding the args.

by the way, this works fine:

```
$ ./target/debug/jd_client -c roles/jd-client/config-examples/proxy-config-local-example.toml
```

Reply:
You're right @plebhash. I'm sorry but I did it quickly yesterday. 
I also tried it and it works as you said.
For me it can be merged 👍